### PR TITLE
Add elemental prefixes

### PR DESCRIPTION
**Pull request by Xe**

### CODE REVIEW

**Comment by DanielOaks** [comment]
Thanks for the PR!

I just added some commits to `chanmembers`. I think I'd rather handle it [this way](https://github.com/DanielOaks/irc-defs/compare/e83f0a7...7329603). I've been meaning to add that note to PROTECTED for a while now, and I think this way prevents unnecessary duplication.

Thoughts @dequis ?

File: _data/chanmembers.yaml [hunk 1]
```diff
@@ -129,6 +138,15 @@ values:
             users.
 
     -
+        name: FOUNDER
+        prefixchar: "~"
+        mode: y
+        origin: Elemental-IRCd
+        comment: >
+          Exactly similar to the above FOUNDER definition, but chosen as
```
Comment (dequis) [inline _data/chanmembers.yaml:146]: > Exactly similar

**Comment by dequis** [comment]
@DanielOaks How about explicit pairs of [mode, prefix]?

Like

```
- ['a', '&']
- ['a', '!']
```

and 

```
- ['q', '~']
- ['y', '~']
```

I think it's better to make this explicit instead of implying that all possible combinations of prefixes and modes may exist.

**Comment by DanielOaks** [comment]
Hmm, I think that could make sense on the YAML side. Just need to make it actually display nicely on the site.

Thinking possibly something like this, with all the modes listed in the same row as the name, just on new lines (blame Github markdown for them being in separate rows).

| Name | Mode |
| --- | --- |
| PROTECTED | `(a)&` |
|  | `(a)!` |
| OWNER | `(q)~` |
|  | `(y)~` |

ie, similar to how they're displayed in the `PREFIX` token on ISUPPORT.

**Comment by dequis** [comment]
:+1:

**Comment by DanielOaks** [comment]
We'll move that to #6, and I'll close this PR for now.

Definitely feel free to open up another PR with any other custom numerics/etc Elemental uses. Always looking for more info. Thanks again for opening it!

**Pull request closed without merging**